### PR TITLE
LPS-200552

### PR DESCRIPTION
**Pull request by TamyrisTorres**
Ticket:https://liferay.atlassian.net/browse/LPS-200552

### CODE REVIEW

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Comment by TamyrisTorres** [comment]
ci:test:hotel

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeSCSSNodeSassPatternsCheck.java [hunk 1]
```diff
@@ -76,22 +76,22 @@ private String _formatInterpolation(Matcher matcher) {
...
-				sb.append(" + '");
-				sb.append(variableName);
-			}
-			else {
-				sb.append(" + '");
-				sb.append(interpolation);
-				sb.append("'}");
+			String[] interpolations = interpolation.split("((\\#\\{)|(\\}))");
+
+			for (String variableName : interpolations) {
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeSCSSNodeSassPatternsCheck.java:81]: @TamyrisTorres, consistency:
```
String[] interpolationParts = interpolation.split("((\\#\\{)|(\\}))");

for (String interpolationPart : interpolationParts) {
```

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeSCSSNodeSassPatternsCheck.java (hunk 1 above)
Comment (TamyrisTorres) [reply modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeSCSSNodeSassPatternsCheck.java]: Resolved

**Review by natocesarrego — APPROVED** [review]

**Comment by natocesarrego** [comment]
ci:test:sf

**Comment by TamyrisTorres** [comment]
ci:test:hotel

**Pull request closed without merging**